bundle/config/artifact.go

```diff
@@ -22,11 +29,10 @@ type Artifact struct {
 	// for example, where setup.py is for Python projects
 	Path string `json:"path"`
 
-	// The relative or absolute path to the built artifact file
+	// The relative or absolute path to the built artifact files
 	// (Python wheel, Java jar and etc) itself
-	File         string `json:"file"`
-	BuildCommand string `json:"build"`
-	RemotePath   string `json:"-" bundle:"readonly"`
+	Files        []ArtifactFile `json:"files"`
```

Comment:
What's the rationale for a slice here? For Python wheels I would expect only a single one. Future proofing?

Reply:
@pietern while discussing this with @lennartkats-db last week we mentioned that it might be multiple output files for a single artifact especially for Java projects. Since the structure is designed to support both, that's the reason for slice.

For Python project I can imagine we might have some setup for monorepo and custom builds which yields multiple Python wheels.